logging/request_context_logger.go

```diff
@@ -0,0 +1,70 @@
+package logging
+
+import (
+	"github.com/giantswarm/request-context"
+)
+
+// NewRequestContextLogger returns a RequestContextLogger, given a Config.
+func NewRequestContextLogger(config Config) RequestContextLogger {
```

Comment:
Arguably, we could remove it.

I'm tempted not to, as there is some useful setup / formatting (see: https://github.com/giantswarm/request-context/blob/master/request_context.go#L46 and https://github.com/giantswarm/request-context/blob/master/request_context.go#L124), which I don't love the idea of basically copying.